src/pomelo_plug.c

```diff
@@ -1,3 +1,5 @@
+// NOTE: This code is based on https://password-hashing.net/submissions/POMELO-v3.tar.gz archive.
+
 // PHC submission:  POMELO v2
```

Comment:
Does the code from the v3 tarball also still have "v2" in this comment, or do we need to update the comment?

Reply:
It still has the "v2" in the comment.